projects/core/src/styles/theme.dark.scss

```diff
@@ -179,6 +179,8 @@
   --cds-alias-viz-severity-warning-light: #{$cds-global-color-yellow-300};
   --cds-alias-viz-severity-immediate: #{$cds-global-color-tangerine-400};
   --cds-alias-viz-severity-critical: #{$cds-global-color-red-700};
+  --cds-alias-viz-severity-free-space-fill: #{$cds-global-color-construction-800};
+  --cds-alias-viz-severity-free-space-border: #{$cds-global-color-construction-400};
```

Comment:
Border is the same for light and dark themes?

Reply:
I picked this up from the story [CDE-2457](https://vmw-jira.broadcom.net/browse/CDE-2457)
_--cds-alias-viz-severity-free-space-border to be linked to:
Light theme: **--cds-global-color-construction-400**
Dark theme: -**-cds-global-color-construction-400**_

Reply:
Since the border is same for light and dark, removing the token from dark theme file.